pkg/fission-cli/package.go

```diff
@@ -36,10 +37,8 @@ import (
 
 	"github.com/dchest/uniuri"
 	"github.com/fission/fission/pkg/utils"
-	"github.com/hashicorp/go-multierror"
 	"github.com/mholt/archiver"
 	"github.com/pkg/errors"
-	"github.com/satori/go.uuid"
 	"github.com/urfave/cli"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
Looks like the imports were missed deleted, please rollback the change.

Reply:
thank @life1347  for your review , that is my fail and I will fix this